Formula/haskell-language-server.rb

```diff
@@ -73,7 +73,7 @@ def caveats
     ghcs.each do |ghc|
       with_env(PATH: "#{ghc.bin}:#{ENV["PATH"]}") do
         assert_match "Completed (1 file worked, 1 file failed)",
-          shell_output("#{bin}/haskell-language-server-#{ghc.version.major_minor} #{testpath}/*.hs 2>&1", 1)
+          shell_output("#{bin}/haskell-language-server-#{ghc.version.major_minor_patch} #{testpath}/*.hs 2>&1", 1)
```

Comment:
```suggestion
          shell_output("#{bin}/haskell-language-server-#{ghc.version.major_minor} #{testpath}/*.hs 2>&1", 1)
```
We don't need this if you apply my suggestion above.